src/Infrastructure/Entity/Entity.php

```diff
@@ -799,7 +799,7 @@ public function save(bool $updateFingerPrint = true): bool
                             $queryParams[] = $value;
                         }
                     }
-                    // Ignore the usr_id_create and timestamp_create (and *_change) columns in the change log...
+                    // Ignore the usr_id_create and timestamp_crearte (and *_change) columns in the change log...
```

Comment:
@kainhofer this line should be reverted

Reply:
Right. Unfortunately, that typo also made it into the other PR, which is already merged, so I added the revert here.